static/styles/user_status.css

```diff
@@ -1,58 +1,11 @@
-.user_status_overlay {
-    .overlay-content {
-        width: 384px;
-        margin: 0 auto;
-        position: relative;
-        top: calc((30vh - 50px) / 2);
-        border-radius: 4px;
-        overflow: hidden;
-        background-color: hsl(0, 0%, 100%);
-
-        @media (width < $ml_min) {
-            width: 100%;
-        }
-    }
-
+#set_user_status_modal {
     input.user_status {
-        width: 336px;
+        width: 95%;
         @media (width < $ml_min) {
-            width: 94%;
+            width: 92%;
         }
     }
 
-    .user-status-header {
-        padding-top: 4px;
-        padding-bottom: 4px;
-        height: 5%;
-        width: 100%;
-        text-align: center;
-        border-bottom: 1px solid hsl(0, 0%, 87%);
-    }
-
-    .user-status-header h1 {
-        margin: 0;
-        font-size: 1.5em;
-    }
-
-    .user-status-header .exit {
-        font-weight: 400;
-        position: absolute;
-        top: 10px;
-        right: 10px;
-        color: hsl(0, 0%, 67%);
-        cursor: pointer;
-    }
-
-    .user-status-header .exit-sign {
-        position: relative;
-        top: 1px;
-        margin-left: 3px;
-        font-size: 1.5rem;
-        line-height: 1;
-        font-weight: 600;
-        cursor: pointer;
-    }
-
     .user-status-options {
         padding: 0 20px 3px;
```

Comment:
This padding needs to be reduced by 15px, since `modal-body` already has some padding, to preserve the design.